### PR TITLE
TW-1996-removed-activated-status-in-accounts-tab

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
 #1996 

## Solution
According to the [figma design](https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=25783-37873&t=2LUxJI6eCDu1cVUk-0 ) only the Not-Activated status should be kept

## Impact description
 the activated status in contacts tab is removed

## Test recommendations
**Recommendations for how to test this, or anything else you are worried about?**

## Resolved
**Attach screenshots or videos demonstrating the changes**
- Web:

https://github.com/user-attachments/assets/a5696b73-e888-4d16-bb37-398a85fc50c8

- Android:

[contacts_demo1.webm](https://github.com/user-attachments/assets/77994f9a-ff3a-431e-afcd-6f41bc8d32c4)

- IOS:

### CODE REVIEW

File: lib/pages/new_private_chat/widget/contact_status_widget.dart [hunk 1]
```diff
@@ -26,21 +25,14 @@ class ContactStatusWidget extends StatelessWidget {
           SvgPicture.asset(
             ImagePaths.icStatus,
             // ignore: deprecated_member_use
-            color: status == ContactStatus.active ? activeColor : inactiveColor,
+            color: inactiveColor,
```
Comment (nqhhdev) [inline lib/pages/new_private_chat/widget/contact_status_widget.dart:28]: Pls use `colorFilter: ColorFilter.mode()` and remove ignore

File: lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart [hunk 2]
```diff
@@ -85,7 +85,7 @@ class ExpansionContactListTile extends StatelessWidget {
                             ),
                           ],
                           const SizedBox(width: 8.0),
-                          if (contact.status != null)
+                          if (contact.status == ContactStatus.inactive)
```
Comment (nqhhdev) [inline lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart:88]: What happens when contact status is null

Comment (KhaledNjim) [reply lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart]: Added null check

File: lib/pages/new_private_chat/widget/contact_status_widget.dart (hunk 1 above)
Comment (KhaledNjim) [reply lib/pages/new_private_chat/widget/contact_status_widget.dart]: done

File: lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart [hunk 3]
```diff
@@ -85,7 +85,8 @@ class ExpansionContactListTile extends StatelessWidget {
                             ),
                           ],
                           const SizedBox(width: 8.0),
-                          if (contact.status != null)
+                          if (contact.status != null &&
+                              contact.status == ContactStatus.inactive)
```
Comment (hoangdat) [inline lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart:89]: Please write widget test for this. Thanks

**Comment by Bobpodvalnyi** [comment]
image where did the separator lines on the contact screen go?

**Comment by KhaledNjim** [comment]
> image where did the separator lines on the contact screen go?

This ticket is only related to account status, the design updates  will be implemented in a seperate ticket

File: test/pages/new_private_chat/widget/contact_status_widget_test.dart [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+void main() {
+  group('ContactStatusWidget', () {
+    testWidgets('renders correctly for inactive status',
+        (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          localizationsDelegates: L10n.localizationsDelegates,
+          supportedLocales: L10n.supportedLocales,
+          home: Scaffold(
+            body: ContactStatusWidget(status: ContactStatus.inactive),
```
Comment (nqhhdev) [inline test/pages/new_private_chat/widget/contact_status_widget_test.dart:18]: Pls add test case for `active`

Comment (KhaledNjim) [reply test/pages/new_private_chat/widget/contact_status_widget_test.dart:18]: done

**Comment by hoangdat** [comment]
LGTM, please update the video demo @KhaledNjim

**Comment by KhaledNjim** [comment]
> LGTM, please update the video demo @KhaledNjim

updated

**Review by hoangdat — APPROVED** [review]

**Pull request merged**